content/docs/workflows/_index.md

```diff
@@ -50,12 +50,19 @@ On the first boot, the Worker is PXE booted, asks Boots for it's IP address, and
 
 If there are no workflows defined for the Worker, the Provisioner will ignore the Worker's request. If as a part of the workflow, a new OS is installed and completes successfully, then the boot request (after reboot) will be handled by newly installed OS. If as a part of the workflow, an OS is **not** installed, then the Worker after reboot will request PXE-boot from the Provisioner.
 
-You can view the events and the state of a workflow during or after its execution with the tink CLI using the [`tink workflow events`](/cli-reference/workflow/#tink-workflow-events) an the [`tink workflow state`](/cli-reference/workflow/#tink-workflow-state) commands.
+You can view the events and the state of a workflow during or after its execution with the tink CLI using the [`tink workflow events`](/docs/cli-reference/workflow/#tink-workflow-events) an the [`tink workflow state`](/docs/cli-reference/workflow/#tink-workflow-state) commands.
 
 ## Ephemeral Data
 
-The workers that are part of a workflow might need to share data. This can take the form of a light JSON like below, or some binary files that other workers might require to complete their action.
-For instance, a worker may add the following data:
+Ephemeral data is data that is shared between Workers as they execute workflows. Ephemeral data is stored in `/workflow/<workflow_id>`.
+
+Initially the directory is empty; you populate with it by having a [template's actions (scripts, etc)](/docs/templates) write to it.
+
+As the workflow progresses, subsequent actions on a Worker can read any ephemeral data that's been created by previous actions on other Workers as well as update that file with any changes. After the workflow finishes, any future actions on additional Workers will be able to read from that directory and see the updates.
```

Comment:
Ephemeral data is only preserved through the life of a single workflow. Each workflow that executes gets a fresh start, i.e. an empty file.

Reply:
Updated, mostly with your wording. Thanks.